client/src/components/Login/Login.js

```diff
@@ -41,7 +41,6 @@ class Login extends React.Component {
       </div>
     );
   }
-
```

Comment:
Let's add this line back in. We don't want to modify files that don't need to be so we can keep the diff clean.

Reply:
Resolved